TrackerMC/inc/StrawDigiBundle.hh

```diff
@@ -0,0 +1,35 @@
+// Bundle together references to various StrawDigi products, to circumvent
+// triplicated-loops which pop up when iterating over digis
+//
+// Ed Callaghan, 2023
+
+#ifndef TrackerMC_StrawDigiBundle_hh
+#define TrackerMC_StrawDigiBundle_hh
+
+// mu2e
+#include <Offline/RecoDataProducts/inc/StrawDigi.hh>
+#include <Offline/MCDataProducts/inc/StrawDigiMC.hh>
+
+namespace mu2e{
+  class StrawDigiBundle{
+    public:
+      StrawDigiBundle(const StrawDigi digi, const StrawDigiADCWaveform adcs, const StrawDigiMC mc);
+      StrawDigiBundle(const StrawDigi digi, const StrawDigiADCWaveform adcs);
+      StrawDigiBundle(const StrawDigiBundle& bundle);
+
+      const StrawDigi GetStrawDigi() const;
```

Comment:
Usually we would return a const reference, so the user can decide if they need to make a copy.

Reply:
The three accessors now return references.